### PR TITLE
RDoc-2948 & RDoc-2949 Cloud Cluster Health documentation

### PR DESCRIPTION
**Pull request by PFYasu**
https://issues.hibernatingrhinos.com/issue/RDoc-2949/Cloud-Cloud-Portal-Products-Tab-Add-Cluster-Health
https://issues.hibernatingrhinos.com/issue/RDoc-2948/Cloud-Cloud-Maintenance-Troubleshooting-Add-Cluster-Health

### CODE REVIEW

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-maintenance-troubleshooting.markdown [hunk 1]
```diff
@@ -25,4 +25,80 @@ status.
...
+
+## Suggestions
+
+This section displays suggestions for a selected *cluster node* based on incident trends.
+
+!["Cluster Health: Suggestions section"](images\cluster-health-suggestions.png "Cluster Health: Suggestions section")
+
+Suggestions are generated based on a 60-day period divided in half.  
+
+**First period** (hereinafter referred to as **previous period**) lasts from *-60* to *-30* days from now.
```
Comment (hiddenshadow21) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-maintenance-troubleshooting.markdown:70]:
```suggestion
**First period** (hereinafter referred to as **previous period**) lasts from *now - 60 days* to *now - 30 days.*
```

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-maintenance-troubleshooting.markdown [hunk 2]
```diff
@@ -25,4 +25,80 @@ status.
...
+## Suggestions
+
+This section displays suggestions for a selected *cluster node* based on incident trends.
+
+!["Cluster Health: Suggestions section"](images\cluster-health-suggestions.png "Cluster Health: Suggestions section")
+
+Suggestions are generated based on a 60-day period divided in half.  
+
+**First period** (hereinafter referred to as **previous period**) lasts from *-60* to *-30* days from now.  
+**Second period** (hereinafter referred to as **current period**) lasts from *-30* days from now to *now*.
```
Comment (hiddenshadow21) [inline Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-maintenance-troubleshooting.markdown:71]:
```suggestion
**Second period** (hereinafter referred to as **current period**) lasts from *now - 30 days* to *now*. 
```

**Comment by hiddenshadow21** [comment]
After the merge of https://github.com/ravendb/docs/pull/1895 you might need to move your files to `Documentation/6.0/...`

**Comment by PFYasu** [comment]
> After the merge of #1895 you might need to move your files to `Documentation/6.0/...`

Will do. Thanks!

File: Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-maintenance-troubleshooting.markdown (hunk 2 above)
Comment (PFYasu) [reply Documentation/4.2/Raven.Documentation.Pages/cloud/cloud-maintenance-troubleshooting.markdown:71]: Good suggestion. Changed.
image

**Review by hiddenshadow21 — APPROVED** [review]

**Pull request merged**